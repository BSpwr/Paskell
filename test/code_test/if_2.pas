program If2Test;
var
   grade: string = 'A';
begin

    grade := 'F';

   if grade = 'B' then
      begin
      writeln('good');
      writeln('second statement of compound')
      end
   else if grade = 'A' then
      writeln('WOW A!')
   else if grade <> 'F' then
      writeln('Not Fail!')
   else begin
        writeln('Compound statement 1');
        writeln('Compound statement 2')
   end;

end.