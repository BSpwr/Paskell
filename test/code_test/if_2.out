Compound statement 1
Compound statement 2